### PR TITLE
[feat]: add f1 reweighted loss for multi tower model & make odpsRtpInput support extra input column, like 'sample_weight

### PR DESCRIPTION
**Pull request by yangxudong**

### CODE REVIEW

File: easy_rec/python/model/multi_tower.py [hunk 1]
```diff
@@ -79,6 +84,14 @@ def build_loss_graph(self):
         loss_value = tf.losses.sigmoid_cross_entropy(labels, logits,
                                                      self._sample_weight)
         self._loss_dict['sigmoid_loss'] = loss_value * loss.weight
+      elif loss.loss_type == LossType.F1_REWEIGHTED_LOSS:
+        if not self._f1_reweight_loss:
+          raise Exception("f1 reweight loss must be configured in model config")
```
Comment (poson) [inline easy_rec/python/model/multi_tower.py:89]: 增加一个github 的config 的链接？

File: easy_rec/python/model/rank_model.py [hunk 2]
```diff
@@ -103,7 +103,7 @@ def _build_loss_impl(self,
                        num_class=1,
                        suffix=''):
     loss_dict = {}
-    if loss_type == LossType.CLASSIFICATION:
+    if loss_type in [LossType.CLASSIFICATION, LossType.F1_REWEIGHTED_LOSS, LossType.PAIR_WISE_LOSS]:
       loss_name = 'cross_entropy_loss' + suffix
```
Comment (poson) [inline easy_rec/python/model/rank_model.py:107]: 都写成cross entropy loss的loss_name ，是不是比较奇怪

Comment (yangxudong) [reply easy_rec/python/model/rank_model.py:107]: 已修改

File: docs/source/models/multi_tower.md [hunk 3]
```diff
@@ -77,6 +80,7 @@ model_config: {
 - losses: 可选，可以选择同时配置两个loss函数，并且为每个loss配置不同的权重
   - loss_type: CLASSIFICATION [默认值] 二分类的sigmoid cross entropy loss
   - loss_type: PAIR_WISE_LOSS [可选] 以优化AUC为主要目标的 pairwise rank loss
+  - loss_type: F1_REWEIGHTED_LOSS [可选] 可以调节二分类模型recall/precision相对权重的loss; 注意不要与`loss_type: CLASSIFICATION`同时使用
```
[the remark this answers is not in the document]
Comment (yangxudong) [reply docs/source/models/multi_tower.md:83]: f1_beta_square 的介绍已添加；
selected_cols: 的说明已添加

**Review by poson — APPROVED** [review]
